RELEASE.md

```diff
@@ -6,6 +6,12 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
+# Release 6.1.1
+
+## Bug fixes and other changes
+
+- Update API response headers for added application security. (#1355)
+
```

Comment:
hey @tynandebold I just realised I forgot to include the disable graphql in the release note. Do you mind putting a line here as well, please? 
```suggestion
- Disable graphiql playground for application security. (#1349)
```

Reply:
I don't think we need to add that in because it's not a user-facing change. I only added this API line so we could have something to add to the `6.1.1` section.

Reply:
okay if so then all good 👍